pkg/client-sdk/ark_sdk.go

```diff
@@ -22,33 +22,24 @@ type ArkClient interface {
 	Balance(ctx context.Context, computeExpiryDetails bool) (*Balance, error)
 	Receive(ctx context.Context) (onchainAddr, offchainAddr, boardingAddr string, err error)
 	SendOffChain(
-		ctx context.Context, withExpiryCoinselect bool, receivers []Receiver,
-		withZeroFees bool,
+		ctx context.Context, withExpiryCoinselect bool, receivers []Receiver, withZeroFees bool,
```

Comment:
```suggestion
		ctx context.Context, withExpiryCoinselect bool, receivers []Receiver
```
We forgot to remove this `withZeroFees` (now it's always zero fees thanks to P2A)